src/Public/LogRhythm/Case/Test-LrProcedureIdFormat.ps1

```diff
@@ -0,0 +1,67 @@
+using namespace System
+using namespace System.IO
+using namespace System.Collections.Generic
+
+Function Test-LrProcedureIdFormat {
```

Comment:
Since Procedure Id can ONLY be a `guid`, this function can be removed.  

You can use `Test-Guid` anywhere you currently use `Test-LrProcedureIdFormat`.  The only result of `Test-LrProcedureIdFormat` would be guid is true or guid is false.  :)

Reply:
So this is more of a design Q.  The Test-LrProcedureIdFormat checks if its a Guid, but it also checks if its an integer.  If its a guid, all is good.  If its an integer, then the procedure is being referenced by its position, ie the 4th procedure on playbook X.  Last, if it's not guid or int, then its treated as a string for doing procedure lookup by name match.

Thoughts?

Reply:
Agree with you about this being a design thing.  Let's leave as-is for now.